apps/src/gamelab/AnimationTab/AnimationTab.jsx

```diff
@@ -86,4 +91,10 @@ var AnimationTab = React.createClass({
     );
   }
 });
-module.exports = Radium(AnimationTab);
+module.exports = connect(state => ({
+  columnSizes: state.animationTab.columnSizes
+}), dispatch => ({
+  onColumnWidthsChange: (widths) => {
+    dispatch(setColumnSizes(widths));
+  }
```

Comment:
Also a nice catch.  I've actually realized since these _have_ to be named, they're probably better expressed with the compact object-of-functions syntax:

```javascript
dispatch => ({
  onColumnWidthsChange(widths) {
    dispatch(setColumnSizes(widths));
  }
})
```

I'll update all my other connect() calls in the next ES6ify PR.